conda_build/inspect_pkg.py

```diff
@@ -96,9 +95,6 @@ def __str__(self):
 untracked_package = _untracked_package()
 
 
-@deprecated.argument("24.1.0", "24.3.0", "platform", rename="subdir")
-@deprecated.argument("24.1.0", "24.3.0", "prepend")
-@deprecated.argument("24.1.0", "24.3.0", "minimal_hint")
```

Comment:
The removal of the arguments had already happened?

Reply:
Correct, the decorator only raises a warning if the argument was passed in